### DISCUSSION TITLE
Add support for AMD PCnet ISA+ or Realtek RTL8019AS?

### DISCUSSION THREAD

**Discussion by Fubuki0x10DE** [joncampbell123/dosbox-x#4709] [General]
These are ISA network cards that support PnP.

**Comment by rderooy** [comment]
There is already an Ethernet discussion: https://github.com/joncampbell123/dosbox-x/discussions/4698

**Comment by RBTCHENTPE** [comment]
Dear Sir,

 

Would please kindly teach me how to modify the keyboard configuration for following problems?

 

windows 10 with DosBox-x the keyboard auto. create numbers of the same letter/digit continuously.

 

B. RGDS.,

ROBERT CHEN
